src/lib/comparison.ts

```diff
@@ -21,10 +45,18 @@ export class Comparison<Location> {
    * @param tokenizer A tokenizer for the correct programming language
    * @param hashFilter An optional HashFilter to filter the hashes returned by
    * the rolling hash function.
+   * @param noFilter A NoFilter used to generate hashes for blacklisted files.
+   * @param passageOptions The options used to filter based on the passage count. For a more detailed explanation see
+   * [[ComparisonOptions]]. If this options is not used then no filtering will occur.
    */
-  constructor(tokenizer: Tokenizer<Location>, hashFilter?: HashFilter) {
+  constructor(tokenizer: Tokenizer<Location>, options?: ComparisonOptions) {
```

Comment:
if options only contains 3 possible values, use object destructuring in the argument list.